### PR TITLE
AF-2766: Dashbuilder Prometheus data-set provider

### PR DESCRIPTION
**Pull request by jesuino**
TBD

Part of an ensemble. Merge with:

https://github.com/kiegroup/kie-soup/pull/172
https://github.com/kiegroup/appformer/pull/1091

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4624,6 +4625,12 @@
         <version>${version.javax.xml.stream.stax}</version>
       </dependency>
 
+      <dependency>
+        <groupId>link.bek.tools</groupId>
+        <artifactId>issue-keeper-junit</artifactId>
+        <version>${version.link.bek.tools.issue-keeper-junit}</version>
+      </dependency>
```
Comment (caponetto) [inline pom.xml:4632]: Out of curiosity, can you explain why this dependency is needed? I guess I haven't seen it being directly used.

File: pom.xml [hunk 2]
```diff
@@ -308,6 +308,7 @@
     <version.com.google.testing.compile>0.11</version.com.google.testing.compile>
     <version.jakarta-regexp>1.4</version.jakarta-regexp>
     <version.jakarta.json.bind>1.0.2</version.jakarta.json.bind>
+    <version.link.bek.tools.issue-keeper-junit>4.11.1</version.link.bek.tools.issue-keeper-junit>
```
Comment (caponetto) [inline pom.xml:311]: Can we use the newest version? `4.12.5`

File: pom.xml [hunk 3]
```diff
@@ -94,7 +94,7 @@
     <version.com.neovisionaries>1.27</version.com.neovisionaries>
     <version.com.networknt.json-schema-validator>1.0.43</version.com.networknt.json-schema-validator>
     <version.com.sun.xml.bind>2.3.0</version.com.sun.xml.bind>
-    <version.com.thoughtworks.xstream>1.4.15</version.com.thoughtworks.xstream>
+    <version.com.thoughtworks.xstream>1.4.14</version.com.thoughtworks.xstream>
```
Comment (caponetto) [inline pom.xml:97]: Can you explain the reason for this downgrade?

**Review by caponetto — COMMENTED** [review]
Hi @jesuino, thanks for the PR!
I've left some comments, please check if they make sense.

File: pom.xml (hunk 1 above)
Comment (jesuino) [reply pom.xml:4632]: This was all probably due a missing rebase, I will update this PR in a minute

**Comment by jesuino** [comment]
Hello @caponetto  - great findings in your review!

I just updated the branch with the only change that should be done: adding the new project version

**Review by caponetto — APPROVED** [review]

**Comment by ederign** [comment]
please jenkins execute fdb

**Review by ederign — APPROVED** [review]

**Pull request merged**